electronics_model/PriceGetter.py

```diff
@@ -0,0 +1,69 @@
+from typing import List, Tuple, Dict
+
+import edgir
+from edg_core import BaseBackend, CompiledDesign, TransformUtil
+import os
+import csv
+import re
+
+
+class GetPrice:
+    def run(lcsc_part_number: str, quantity: int) -> float:
+        cur_path = os.path.dirname(__file__)
+        parts_library = os.path.relpath(
+            '..\\electronics_lib\\resources\\Pruned_JLCPCB SMT Parts Library(20220419).csv', cur_path)
+
+        price_string: str = 'NA'
+        with open(parts_library, 'r', newline='') as csv_file:
+            csv_reader = csv.DictReader(csv_file, 'r')
+            for row in csv_reader:
+                if row['LCSC Part'] == lcsc_part_number:
+                    price_string = row['Price']
+                    break
+            if price_string == 'NA':
+                return -1
+            csv_file.close()
+
+        temp = price_string.split(',')  # temp = list of all the qty ranges & prices per range
```

Comment:
Style: consider giving this a more descriptive name, maybe prices_by_quantity or something?

Reply:
done